### PR TITLE
Privacy consent

### PR DESCRIPTION
**Pull request by inverse**
@alexesprit started working on this to meet FF toc.

image

### CODE REVIEW

File: src/core/background/main.js [hunk 1]
```diff
@@ -437,7 +437,16 @@ require([
...
+
+		chrome.tabs.create({
+			url: '/startup/startup.html'
+		  });
 	}
 
 	startup();
+
+
+	// chrome.runtime.onInstalled.addListener(function() {
```
Comment (inverse) [inline src/core/background/main.js]: Was thinking of plugging in here for first time users.

**Comment by inverse** [comment]
@alexesprit got a working prototype ready. def could use some design love.

**Comment by inverse** [comment]
It's now pretty usable - just needs localization support but that could come later I guess.

**Comment by flip-bs** [comment]
thank you a lot <3

**Pull request merged**